src/main/java/org/terasology/moduletestingenvironment/ModuleTestingEnvironment.java

```diff
@@ -375,7 +374,7 @@ public void setSafetyTimeoutMs(long safetyTimeoutMs) {
     private TerasologyEngine createHeadlessEngine() throws IOException {
         TerasologyEngineBuilder terasologyEngineBuilder = new TerasologyEngineBuilder();
         terasologyEngineBuilder
-                .add(new WithUnittestModule())
+//                .add(new WithUnittestModule())
```

Comment:
@keturn just introduced this recently - why did you comment it out here?

Reply:
looks like a mistake you can revert this.

Reply:
reverted